sass/index.scss

```
@@ -668,15 +681,4 @@ footer {
             text-align: left;
         }
     }
-}
-
-@media screen and (max-width: 475px) {
-    // Contact
-    #contact #contact-socials {
-        justify-content: flex-start;
-        padding: 0;
-        margin: 2rem 0;
-        background-image: none;
-        height: auto;
-    }
-}
+}
```

nit: missing newline at end of file
```suggestion
}

```